pyvista/_typing.py

```diff
@@ -27,7 +25,7 @@
     Dict[str, Union[int, float, str]],
     str,
     "Color",
-    _vtk.vtkColor3ub,
+    # _vtk.vtkColor3ub,
```

Comment:
```suggestion
```

Reply:
Ah thanks for pointing this out! We may actually want to keep that type in there. I commented it out because it required importing the plotting module and I was trying to get the top level to avoid any plotting imports. I will push some changes that clean this up rather than removing this

Reply:
Cleaned this up

Reply:
Cleaned this up